scripts/ci/local_ci_fix_ownership.sh

```diff
@@ -30,7 +30,7 @@ basic_sanity_checks
 
 script_start
 
-export PYTHON_VERSION=${PYTHON_VERSION:="3.6"}
+export PYTHON_VERSION=3.5
```

Comment:
Does this mean that I will have two images?

Reply:
By default we have python 3.5 used for all static tests - I synchronised it to use 3.5 everywhere (and we do not rely on local python version any more). So you will have just one image (3.5) unless you explicitly choose --python-version= in breeze or set PYTHON_VERSION variable.